build-docker-compose.yml

```diff
@@ -15,3 +15,4 @@ services:
         context: modules/.
         args:
             - MODULES_TAG=${MODULES_TAG}
+            - MODULES_COMMIT=${MODULES_COMMIT}
```

Comment:
This variable is required but it is commented out in `template.env`. Does it have to be mandatory?

Reply:
Ideally the dockerfile check whether it's empty or not, if commit is not empty, it overrides tag.

Reply:
But docker itself would complain. You can add a default value with the notation `${MODULES_COMMIT:-<DefaultValue>}`

Reply:
@Phat3 I am not sure whether this is he case. Look at the Dockerfile inside `server`. We do the same thing and we have no side-effect.

Reply:
I get this when building:
```
WARN[0000] The "MISP_COMMIT" variable is not set. Defaulting to a blank string.
WARN[0000] The "MODULES_COMMIT" variable is not set. Defaulting to a blank string.
```

Reply:
Yep you get warnings though. If you are fine with those we can leave it as is

Reply:
Yes warnings are fine